### ISSUE TITLE
Bluestacks Window Identification

### ISSUE THREAD

**Issue by JartanFTW** [JartanFTW/kot-skipper#12]
If the Bluestacks subwindow has any name other than Bluestacks, program fails to identify it.
I'm thinking that the whole advanced specific window screenshot system I made (stole, with credit) from stackoverflow might be coming back to bite me. It was used for the upside that it screenshots the game even if another window is on-top of it, but really what I wanted was the ability to screenshot the window if it was minimized but I couldn't find any way to do that.
I think it might be a better idea to just screenshot the entire screen and filter through instead. Some sort of identification system still needs to be implemented for resized windows, see #10 and #11

**Comment by JartanFTW** [comment]
It may be worth considering a major functionality change to using android debug bridge: https://github.com/JeffLIrion/adb_shell

Using adb would allow for screenshots of the game while another window is on-top of it also.

**Comment by JartanFTW** [comment]
https://github.com/JeffLIrion/adb_shell/issues/202

**Comment by JartanFTW** [comment]
> It may be worth considering a major functionality change to using android debug bridge: https://github.com/JeffLIrion/adb_shell

Turns out the functionality change really isn't all that major thanks to pretty decent organization (other than everything being in main.py). Only waiting on assistance saving screenshots in memory rather than to files. 
I did try and do some subclassing to do this myself, but adb_shell is dependent upon itself in so many places it makes it difficult, as well as the fact that I simply don't really understand what's going on due to lack of experience with adb.
